docs/specification.md

```diff
@@ -802,3 +799,75 @@ unit
 - Fails with `NOT_MIGRATION_TARGET` if the sender of this call is not the address of the new version.
 
 [FA2]: https://gitlab.com/tzip/tzip/-/blob/3a6464b1e641008b77a83807a0c102e7602c6af4/proposals/tzip-12/tzip-12.md
+
+# How to write DAOs
```

Comment:
Hm, I think it's not the best place for this chapter. Probably users of the contract shouldn't need to read this document, it's mostly for the customer. We also have autodoc for end users that describes entrypoints.

I would put this "How to write DAOs" chapter into "Usage" instructions in README or into a new document if we find it too big for README.